setup.py

```diff
@@ -34,6 +34,12 @@ def find_version(file_path):
 setup(
     name='FlowCal',
 
+    entry_points = {
+        'console_scripts': [
+            'flowcal=FlowCal.excel_ui:run_command_line'
```

Comment:
Why was the entrypoint `FlowCal` eliminated?

Reply:
@renxida raised this issue here: https://github.com/pypa/pip/issues/4771

Reply:
In light of @renxida's post and possible uninstall problems, I think we should just go with lowercase `flowcal`.

Separately, I think it's a good idea to incorporate verification of a clean uninstall in the build process (I hate it when a package sloppily fails to uninstall correctly and pollutes my system).

Reply:
See #281 for a discussion on installation.

Assuming that I install the package with `pip install .`, uninstallation proceeds cleanly, so I'm unable to reproduce this issue.

On FlowCal's folder:

```
$ sudo pip install .
Processing /home/[redacted]/FlowCal
Requirement already satisfied: packaging>=16.8 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python2.7/dist-packages/six-1.11.0-py2.7.egg (from FlowCal==1.1.4)
Requirement already satisfied: numpy>=1.8.2 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: scipy>=0.14.0 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: matplotlib>=1.3.1 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: palettable>=2.1.1 in /usr/local/lib/python2.7/dist-packages/palettable-3.0.0-py2.7.egg (from FlowCal==1.1.4)
Requirement already satisfied: scikit-learn>=0.16.0 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: pandas>=0.16.1 in /usr/local/lib/python2.7/dist-packages (from FlowCal==1.1.4)
Requirement already satisfied: xlrd>=0.9.2 in /usr/local/lib/python2.7/dist-packages/xlrd-1.0.0-py2.7.egg (from FlowCal==1.1.4)
Requirement already satisfied: XlsxWriter>=0.5.2 in /usr/local/lib/python2.7/dist-packages/XlsxWriter-0.9.6-py2.7.egg (from FlowCal==1.1.4)
Requirement already satisfied: pyparsing in /usr/local/lib/python2.7/dist-packages (from packaging>=16.8->FlowCal==1.1.4)
Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python2.7/dist-packages (from matplotlib>=1.3.1->FlowCal==1.1.4)
Requirement already satisfied: python-dateutil in /usr/local/lib/python2.7/dist-packages (from matplotlib>=1.3.1->FlowCal==1.1.4)
Requirement already satisfied: functools32 in /usr/local/lib/python2.7/dist-packages (from matplotlib>=1.3.1->FlowCal==1.1.4)
Requirement already satisfied: pytz in /usr/local/lib/python2.7/dist-packages (from matplotlib>=1.3.1->FlowCal==1.1.4)
Requirement already satisfied: subprocess32 in /usr/local/lib/python2.7/dist-packages (from matplotlib>=1.3.1->FlowCal==1.1.4)
Installing collected packages: FlowCal
  Running setup.py install for FlowCal ... done
Successfully installed FlowCal-1.1.4
```

Somewhere else (not in the FlowCal folder):
```
$ flowcal
$ FlowCal
```

(Both commands work and open the "open file" dialog.

Same location:

```
$ sudo pip uninstall FlowCal
Uninstalling FlowCal-1.1.4:
  /usr/local/bin/FlowCal
  /usr/local/bin/flowcal
  /usr/local/lib/python2.7/dist-packages/FlowCal-1.1.4-py2.7.egg-info
  /usr/local/lib/python2.7/dist-packages/FlowCal/__init__.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/__init__.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/excel_ui.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/excel_ui.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/gate.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/gate.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/io.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/io.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/mef.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/mef.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/plot.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/plot.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/stats.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/stats.pyc
  /usr/local/lib/python2.7/dist-packages/FlowCal/transform.py
  /usr/local/lib/python2.7/dist-packages/FlowCal/transform.pyc
Proceed (y/n)? y
  Successfully uninstalled FlowCal-1.1.4
$ flowcal
bash: /usr/local/bin/flowcal: No such file or directory
$ FlowCal
bash: /usr/local/bin/FlowCal: No such file or directory
```

After starting another bash session:

```
$ flowcal
flowcal: command not found
$ FlowCal
FlowCal: command not found
```

Reply:
I successfully reproduced the problem in python 3.6. Again, after installing with `sudo pip3.6 install .` and confirming that everything works, I get the following:

```
$ sudo pip3.6 uninstall FlowCal
Exception:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/local/lib/python3.6/dist-packages/pip/commands/uninstall.py", line 76, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/usr/local/lib/python3.6/dist-packages/pip/req/req_set.py", line 346, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/usr/local/lib/python3.6/dist-packages/pip/req/req_install.py", line 734, in uninstall
    FakeFile(dist.get_metadata_lines('entry_points.txt'))
  File "/usr/lib/python3.6/configparser.py", line 763, in readfp
    self.read_file(fp, source=filename)
  File "/usr/lib/python3.6/configparser.py", line 718, in read_file
    self._read(f, source)
  File "/usr/lib/python3.6/configparser.py", line 1092, in _read
    fpname, lineno)
configparser.DuplicateOptionError: While reading from '<???>' [line  3]: option 'flowcal' in section 'console_scripts' already exists
```

This doesn't happen if I only have the entrypoint `flowcal`.

It is interesting that this error only happens in python 3.6 (and possibly other versions of python 3). I will remove the entrypoint `FlowCal` as discussed.